ffi_interface/src/lib.rs

```diff
@@ -297,18 +304,18 @@ pub fn create_proof(context: &Context, input: Vec<u8>) -> Result<Vec<u8>, Error>
     );
 
     // - Deserialize proof queries
-    //
     let mut prover_queries: Vec<ProverQuery> = Vec::with_capacity(num_openings);
 
     for proof_bytes in proofs_bytes {
         let prover_query = deserialize_proof_query(proof_bytes);
         prover_queries.push(prover_query);
     }
 
-    // - Create proofs
-    //
+    let transcript_string = context.transcript_string.clone();
+    // This is a dirty fix due to static lifetimes in transcript.rs file.
```

Comment:
this is still WIP, tests are failing, but do you have any comment on this? @kevaundray